site/content/docs/main/data-movement-backup-pvc-configuration.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "BackupPVC Configuration for Data Movement Backup"
+layout: docs
+---
+
+`BackupPVC` is an intermediate PVC created by the exposer for VGDP (Velero Generic Data Path) to access data from during
+the data movement backup operation.
+
+In some scenarios users may need to configure some advanced options of the backupPVC so that the data movement backup
```

Comment:
Let's include the details of the benefits which could be copied from the design.

Reply:
Done, updated